src/App.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import './App.css'
+
+class App extends Component {
+  state = { foodItems: [] }
+
+  componentDidMount () {
+    axios.get('menu.json').then(response => {
+      this.setState({
+        foodItems: response.data
+      })
+    })
+  }
+
+  render () {
+    const foodItems = this.state.foodItems
```

Comment:
please extract this to a separate component

Reply:
Fixed this.